extensions/pkg/controller/worker/machine_crds.go

```diff
@@ -166,6 +166,31 @@ func init() {
 				},
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "machineclasses.machine.sapcloud.io",
+			},
+			Spec: apiextensionsv1beta1.CustomResourceDefinitionSpec{
+				Group:   machineGroup,
+				Version: machineVersion,
+				Scope:   apiextensionsv1beta1.NamespaceScoped,
+				Names: apiextensionsv1beta1.CustomResourceDefinitionNames{
+					Kind:       "MachineClass",
+					Plural:     "machineclasses",
+					Singular:   "machineclass",
+					ShortNames: []string{"machcls"},
+				},
+				AdditionalPrinterColumns: []apiextensionsv1beta1.CustomResourceColumnDefinition{
+					{
+						Name:        "Provider",
+						Type:        "string",
+						JSONPath:    ".provider",
+						Description: "Provider is the combination of name and location of cloud-specific drivers",
```

Comment:
```suggestion
						Description: "Name of the provider which acts on the machine class object",
```
Location no longer applies as the gRPC mechanism is replaced by the controller based driver. Hence just, the provider name should suffice.